docs/level_0/first_simple_example.md

```diff
@@ -0,0 +1,219 @@
+# First simple Example
+
+In this example we describe how to deploy an nginx with the help of Landscaper.
+
+We describe how to develop a component containing a Component Descriptor and a Blueprint with a DeployItem.
```

Comment:
not sure if we should really use the first tutorial here as it covers multiple personas (developer, operator).
Maybe we should keep this first tutorial as easy as possible.
So I propose to only cover the operator part:
- expect that a blueprint is already given. (use the ingress or the aggregated example)
- just install the landscaper and use the given resource to deploy a simple application.

Reply:
we have divided the example in two more simple ones starting with just an installation